adapters/mqtt-vertx/src/main/java/org/eclipse/hono/adapter/mqtt/VertxBasedMqttProtocolAdapter.java

```diff
@@ -222,10 +222,12 @@ public void doStop(final Future<Void> stopFuture) {
 
     void handleEndpointConnection(final MqttEndpoint endpoint) {
 
-        LOG.info("connection request from client {}", endpoint.clientIdentifier());
+        LOG.info("connection request from client [{}]", endpoint.clientIdentifier());
```

Comment:
IMHO we should log this at debug level, otherwise, this will blow up the log ...